lib/zone-spec/fake-async-test.ts

```diff
@@ -86,7 +86,7 @@
     private _microtasks: Function[] = [];
     private _lastError: Error = null;
     private _uncaughtPromiseErrors: {rejection: any}[] =
-        Promise[Zone['__symbol__']('uncaughtPromiseErrors')];
+        Promise[Zone['__symbol__']('uncaughtPromiseErrors')] || [];
```

Comment:
Could you provide a test showing how this could be undefined? I fear that while `|| []` solves the runtime issue it creates another issue which is that the errors will not be reported.